src/components/Popover/Popover.scss

```diff
@@ -39,10 +39,8 @@ $block: '.#{variables.$ns}popover';
             width: 100%;
             margin-top: 20px;
             $buttonClass: #{$class}__tooltip-button;
-
-            #{$buttonClass}:not(:first-child) {
-                margin-left: 10px;
-            }
+            flex-wrap: wrap;
+            gap: 10px;
```

Comment:
This will not work in Safari <14.5

We can add margin: 5px, for `$buttonClass` and -5px for `&-buttons` instead